README.md

```diff
@@ -128,14 +128,24 @@ Assuming you already have everything needed to install the apps, this is what yo
 
 3. Initialize your environment and configuration:
    Note that this will *not* overwrite existing values, but it will append to existing files.
-   See the [ck8s][ck8s] repository if you are uncertain about in what order you should do things.
+   See the [compliantkubernetes][compliantkubernetes] repository if you are uncertain about in what order you should do things.
 
    ```bash
    ./bin/ck8s init
    ```
 
 4. Edit the configuration files that have been initialized in the configuration path.
-   Make sure that the `objectStorage` values are set in `sc-config.yaml`, `wc-config.yaml` and `secrets.yaml` according to your `objectStorage.type` (so `objectStorage.s3.*` if you are using s3 or `objectStorage.gcs.*` if you are using gcs.)
+   Make sure that the `objectStorage` values are set in `sc-config.yaml`, `wc-config.yaml` and `secrets.yaml` according to your `objectStorage.type` (Set `objectStorage.s3.*` if you are using s3 or `objectStorage.gcs.*` if you are using gcs.)
+   If you enable object storage you also need to make sure that the buckets specified in `objecStorage.buckets` exist.
+   You can can running the following snippet to ensure that you've configured S3 correctly:
+
+   ```bash
+   for bucket in $(yq r ${CK8S_CONFIG_PATH}/sc-config.yaml 'objectStorage.buckets.*'); do
```

Comment:
Is this going to use the keys configured in `secrets.yaml`? If not, I see a risk that it will give the user a false sense of safety.

Reply:
Yes of course that is what ./bin/ck8s uses.

Reply:
Gah, missed that it uses s3cfg, will fix.